setup.py

```diff
@@ -43,7 +43,7 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket ~= 1.19",
+        "pytket >= 1.20.0rc2, <2.0",
```

Comment:
Why do we want to make this change?

Reply:
"pytket ~= 1.19" will not install the pre-release, I wanted to have that in here because it includes the wasm fix to make sure this still works with this extension

Reply:
Don't the tests get run on the latest pre-release anyway?

Reply:
Yes, but I wanted to do a release candidate for other people to test this

Reply:
Yes, I thought we might could do a pre release for other people to test, but that is not really needed. They can just install it locally.